src/services/expensesService.js

```diff
@@ -0,0 +1,90 @@
+const { Op } = require('sequelize');
+const { models } = require('../models/models.js');
+const { Expense } = models;
+
+const getAllExpenses = async (userId, categories, from, to) => {
+  const where = {};
+
+  if (userId) {
+    where.userId = userId;
+  }
+
+  if (categories) {
+    where.category = { [Op.in]: categories };
+  }
+
+  const isValidDate = (date) => !isNaN(Date.parse(date));
```

Comment:
Consider moving the `isValidDate` function outside of `getAllExpenses` if it might be reused elsewhere, or even to a utility module if applicable.